docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md

```diff
@@ -66,7 +66,7 @@ A vertex data file consists of multiple rows, with each line in the file represe
 
 #### Edges
 
-An edge data file consists of multiple rows, with each line in the file representing a point and its properties. In general, the first column is the ID of the source vertex, the second column is the ID of the destination vertex. These ID columns are specified in the mapping file. Other columns are the properties of the edge. Consider the following example in JSON format.
+An edge data file consists of multiple rows. Each line in the file represents a point and its properties. In general, the first column is the ID of the source vertex, the second column is the ID of the destination vertex. These ID columns are specified in the mapping file. Other columns are the properties of the edge. Consider the following example in JSON format.
```

Comment:
"represents a point and its properties."==>"represents one point and its properties."

"Other columns" ==> "The other columns"